frequently-asked-questions.md

```diff
@@ -154,7 +161,7 @@ const getDefaultChannel = (guild) => {
   // Now we get into the heavy stuff: first channel in order where the bot can speak
   // hold on to your hats!
   return guild.channels.cache
-   .filter(c => c.type === "text" &&
+   .filter(c => c.type === "TEXT" &&
```

Comment:
```suggestion
   .filter(c => c.type === "GUILD_TEXT" &&
```

Reply:
Fixed in b5c8ebfc63f03855f21e383c7d0c06ddeea78255